server/Makefile

```
@@ -404,11 +404,15 @@ mmctl-mocks: setup-go-work ## Creates mocks for mmctl
 	cd ./public && $(GO) generate $(GOFLAGS) ./plugin
 
+default-roles-permissions: start-docker ## Generates default_roles_permissions.js by snapshotting a live database.
+	$(GO) run $(GOFLAGS) ./scripts/default_permissions_generator \
+		-out ../e2e-tests/cypress/tests/support/api/default_roles_permissions.js
+
 mocks: store-mocks filestore-mocks ldap-mocks plugin-mocks einterfaces-mocks searchengine-mocks sharedchannel-mocks misc-mocks email-mocks platform-mocks mmctl-mocks mocks-public cache-mocks
 
 layers: store-layers pluginapi
 
-generated: mocks layers
+generated: mocks layers default-roles-permissions
```

checkmake (0.3.2)

[warning] Target "generated" should be declared PHONY.

(phonydeclared)